WordleGame/WordleGame/Models/Game/GameManager.swift

```diff
@@ -8,16 +8,24 @@
 import Foundation
 import UIKit
 
-struct GameManager {
+class GameManager {
     
     private var currentLetterIndexInRow = 0
     private var currentAttemptIndex = 0
     
     private var resultWord: String!
     
+    var gameTime = 0
+    var timer: Timer?
+    
+    var delegate: AlertDelegate?
+    var resultDelegate: GameResultDelegate?
```

Comment:
Do you still need delegate? I can't see where you set it

Reply:
Yes, I use both delegates. First one for handle win or lose game. Second to save gameResult